src/providers/ldap/sdap_async_initgroups.c

```diff
@@ -2754,6 +2764,12 @@ struct tevent_req *sdap_get_initgr_send(TALLOC_CTX *memctx,
 
     state->ev = ev;
     state->opts = id_ctx->opts;
+    state->user_map = user_map;
+    state->user_map_cnt =user_map_cnt;
```

Comment:
very minor nitpick, missing space after the `=` character

Reply:
fixed